src/tuner/nccl_ofi_model.c

```diff
@@ -49,19 +49,18 @@ float nccl_ofi_tuner_compute_cost(struct nccl_ofi_tuner_model_params *params, st
 			num_internode_steps = 2 * dims->num_nodes;
 			latency = (num_internode_steps * net_lat)
 				  + (num_steps - num_internode_steps) * p2p_lat;
-			bw = params->internode_bw * params->num_rails * ofi_nccl_tuner_num_channels();
```

Comment:
Credit where it's due: Just remembered @liralon had caught this during an internal review of the tuner, but it got lost.